readme.txt

```diff
@@ -0,0 +1,38 @@
+=== Design Experiments ===
+Contributors: wordpressdotorg, kjellr, karmatosed, mapk, azaozz, andraganescu, melchoyce, tinkerbelly
+Tags: design, experiments
+Stable tag: trunk
+Requires at least: 4.6
+Tested up to: 5.2
+Requires PHP: 5.2.4
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+A plugin for trying out design experiments in WP-Admin. Brought to you by the WordPress Design Team.
+
+== Description ==
+
+The Design Experiments plugin is a tool for prototyping designs within the context of WP-Admin. It contains a number of proof-of-concepts that the [WordPress Design Team](http://make.wordpress.org/design/) is exploring. 
+
+An important disclaimer: This plugin is called experimental for a reason. It is for testing concepts, and is not intended for use on a production site. These experiments may hide or visually break functionality in the admin area. Some of the experiments may break after a while, and will be removed from the plugin.
```

Comment:
Can we add something like, "Do not rely on this plugin for permanent changes to your site." to the end of this paragraph.

Reply:
👍 Added